packages/components/card/README.md

```diff
@@ -534,7 +533,9 @@ The card Nunjucks macro takes the following arguments:
 | **headingLevel**    | integer | No       | Optional heading level for the card heading. Default: 2                                                             |
 | **href**            | string  | No       | The value of the card href attribute                                                                                |
 | **clickable**       | boolean | No       | If set to true, then the class `nhsuk-card--clickable` will be applied.                                             |
+| **arrow**           | boolean | No       | If set to true, then the class `nhsuk-card__content--arrow` will be applied.                                        |
 | **feature**         | boolean | No       | If set to true, then the class `nhsuk-card__heading--feature` and `nhsuk-card__content--feature` will be applied.   |
+| **tranparent**      | boolean | No       | If set to true, then the class `nhsuk-card--transparent` and `nhsuk-card__content--transparent` will be applied.    |
```

Comment:
Spelling - "transparent". Also, couldn't find the transparent classes anywhere. Is this meant to be 'secondary'?

Reply:
The card was initially called 'transparent', but after speaking with the design team, it was changed to 'secondary'; this was missed in the update and updated accordingly.